apis/app_handler_test.go

```diff
@@ -125,48 +129,48 @@ func testAppGetHandler(t *testing.T, when spec.G, it spec.S) {
 				},
 				"links": {
 				  "self": {
-					"href": "https://api.example.org/v3/apps/`+appGUID+`"
+					"href": "%[1]s/v3/apps/`+appGUID+`"
```

Comment:
I like this change, but it seems like you should continue the pattern for appGUID and spaceGUID, since they're both also repeated many times.

Reply:
This should be done now, however, some of the new files that were pulled in from main may not be using this pattern